packages/generator/src/operations/file.ts

```diff
@@ -17,9 +17,7 @@ import { operationFunction } from './operation';
 export function operationsSourceFile(
   service: VdmServiceMetadata
 ): SourceFileStructure {
-  const functions = service.functionImports;
-  const actions = service.actionImports || [];
-  const operations = [...functions, ...actions];
+  const operations = service.operationImports;
```

Comment:
[req] I think this is not necessary anymore: `operationImportDeclarations` should take just one argument and we can easily reach the operations through `service.operationImports`.